pkg/parent/parent.go

```diff
@@ -77,12 +78,33 @@ func checkPreflight(opt Opt) error {
 	return nil
 }
 
+func createCleanupLock(sDir string, tempDir bool) error {
+	//lock state dir when using /tmp/ path
```

Comment:
Does something bad happen if we always lock the dir regardless to whether it is under /tmp or not?

Reply:
nope that should be no problem.
I only skipped that because you didn't need that, but it's simpler and better to understand.